is_prime.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(n):
+    if n <= 1:
+        return False
+    for i in range(2, int(n ** 0.5) + 1):
```

Comment:
Optimize the cycle so that it runs until the square root of n.

Reply:
Changed.

Reply:
The range end should be int(n ** 0.5) + 1.

Reply:
Good catch! Fixed.